src/Umbraco.Core/Umbraco.Core.csproj

```diff
@@ -52,8 +52,8 @@
     <Reference Include="ICSharpCode.SharpZipLib">
       <HintPath>..\packages\SharpZipLib.0.86.0\lib\20\ICSharpCode.SharpZipLib.dll</HintPath>
     </Reference>
-    <Reference Include="ImageProcessor, Version=2.4.5.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\ImageProcessor.2.4.5.0\lib\net45\ImageProcessor.dll</HintPath>
+    <Reference Include="ImageProcessor, Version=2.5.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\ImageProcessor.2.5.0.0\lib\net45\ImageProcessor.dll</HintPath>

```

Comment:
So this doesn't work for me. Nuget restores the package fine but it restores to this folder ImageProcessor.2.5.0 and not to the hintpath ImageProcessor.2.5.0.0. Can't really figure out why that is.